rules/integrations/aws/credential_access_new_terms_secretsmanager_getsecretvalue.toml

```diff
@@ -113,7 +115,7 @@ reference = "https://attack.mitre.org/tactics/TA0006/"
 
 [rule.new_terms]
 field = "new_terms_fields"
-value = ["aws.cloudtrail.user_identity.access_key_id", "aws.cloudtrail.request_parameters"]
+value = ["user.id", "aws.cloudtrail.request_parameters"]
```

Comment:
Do we have any way to make it unique along instances? Similar to specifying host.id or agent.id for endpoint events.